Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -1666,6 +1986,7 @@ static void loadPrimitiveGameThreadPart(
   // mesh or not. We don't want the editor creating collision meshes itself in
   // the game thread, because that would be slow.
   pBodySetup->bCreatedPhysicsMeshes = true;
+  pBodySetup->bSupportUVsAndFaceRemap = true;
```

Comment:
I think users have to enable UV results from line traces as a project wide setting and then it shows up as part of the HitResult struct. If the above option is set to true and if the project wide setting is set to true, then I think the physX cooking saves the UV coordinates for line traces later. This will take up more CPU memory, but I doubt it should affect cook time (UVs are not needed for collision detection, so BVH insertion or whatever spatial pre-processing is happening will probably not do anything special with the UVs). Later line traces will have to do a simple UV interpolation, but that is trivial as well. 

So TLDR; the cost is just extra CPU memory for the physX mesh.